src/istio/control/http/attributes_builder.h

```diff
@@ -52,8 +54,7 @@ class AttributesBuilder {
   // not available.
   void ExtractAuthAttributes(CheckData* check_data);
 
-  // The request context object.
-  RequestContext* request_;
+  istio::mixer::v1::Attributes* attributes_;
```

Comment:
Could you add a `TODO` to get rid of this raw pointer?

Reply:
happily